package.nls.json

```diff
@@ -0,0 +1,68 @@
+{
+  "extension.runQuery" : "Execute Query",
```

Comment:
Is there an xliff file for these? And a process to build the localized package.nls files during build?